### ISSUE TITLE
_importer.py question

### ISSUE THREAD

**Issue by lmorabit** [revoltek/losoto#38]
I'm updating some python scripts from the prefactor pipeline to use the newest version of losoto.  Is there a reason that the importer.py module is now _importer.py?

**Comment by revoltek** [comment]
Yes, the underscore prevents it from being listed in the documentation that is automatically generated

**Comment by mhardcastle** [comment]
Is there a more fundamental reason why this interface to importing is discouraged, though? Seems like a backward step.

**Comment by lmorabit** [comment]
It also breaks some of the prefactor scripts.  I've updated them, but will
this change in the future?

On Wed, Mar 21, 2018 at 7:19 AM, Martin Hardcastle <notifications@github.com

**Comment by revoltek** [comment]
Yes, on the longer term the importer will become obsolete as NDPPP now does read/write of h5parm directly, so there's no reason for a parmdb<->h5parm convertion anymore

**Comment by lmorabit** [comment]
I would argue that there's no reason for a parmdb <-> h5parm conversion
*only* when all pipelines have been updated to use the new functionality of
NDPPP.  Please don't remove this module without a discussion with the
community.
